### PR TITLE
Fixed how to specify the namespace in `Reset` function for resources with a namespace

### PR DESCRIPTION
**Pull request by 196Ikuchil**
#### What type of PR is this?

/kind bug
#### What this PR does / why we need it:
Fixed how to specify the namespace in `Reset` function for resources with a namespace.
The current Reset function can't work correctly.
I'm so sorry, this is caused by my fixing of #116.

#### Which issue(s) this PR fixes:

Relates #84, #116

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

**Comment by 196Ikuchil** [comment]
This fix will work correctly in the default namespace.
Other namespaces continue to have bugs in their behavior. See also #84.

/assign @sanposhiho

File: node/node.go [hunk 1]
```diff
@@ -101,16 +101,23 @@ func (s *Service) DeleteCollection(ctx context.Context, lopts metav1.ListOptions
...
 		eg.Go(func() error {
 			// delete pods on specific node
 			lopts := metav1.ListOptions{
 				FieldSelector: "spec.nodeName=" + n.Name,
 			}
-			// This method deletes all pods on all namespaces scheduled to the specified node.
-			if err := s.podService.DeleteCollection(ctx, metav1.NamespaceAll, lopts); err != nil {
-				return xerrors.Errorf("failed to delete pods on node %s: %w\n", n.Name, err)
+			for _, ns := range nsList.Items {
+				ns := ns
```
Comment (sanposhiho) [inline node/node.go:116]: Not needed?
```suggestion
```

File: node/node.go [hunk 2]
```diff
@@ -101,16 +101,23 @@ func (s *Service) DeleteCollection(ctx context.Context, lopts metav1.ListOptions
...
+				// This method deletes all pods on specified namespace scheduled to the specified node.
```
Comment (sanposhiho) [inline node/node.go:117]: move this comment to L115(before `for _, ns := range nsList.Items {`)

**Review by sanposhiho — COMMENTED** [review]
just small nits

**Comment by 196Ikuchil** [comment]
@sanposhiho updated/

**Review by sanposhiho — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**